packages/webchat/src/main.tsx

```diff
@@ -88,19 +95,30 @@ class Web extends React.Component<MainProps> {
   async load() {
     this.config = this.extractConfig()
 
+    // TODO: 6 - this implies iframe as well. Do we need this? Why?
     if (this.config.exposeStore) {
       const storePath = this.config.chatId ? `${this.config.chatId}.webchat_store` : 'webchat_store'
       set(window.parent, storePath, this.props.store)
     }
```

Comment:
I think we could remove it, because exposing the store doesnt work across domains (iframe vs parent frame). But lets leave it for now.